src/test/java/it/pagopa/pn/paperchannel/integrationtests/Paper_RS_AR_IT.java

```diff
@@ -418,7 +424,10 @@ void Test_AR_RefusedToStorage__RECRN004A_RECRN004B_RECRN004C() {
         CommonMetaDematAggregateSequenceTest("RECRN004A", "RECRN004B", "RECRN004C", StatusCodeEnum.KO, false, false);
     }
 
+
+    /* Da eliminare perchè sono state implementati in PN-5972 */
     @Test
+    @Disabled
     void Test_AR_CompletedStorage__RECRN005A_RECRN005B_RECRN005C() {
         // meta, demat, final (send to delivery push)
         //
```

Comment:
Andrebbero comunque aggiornate le sequenze in base alle modifiche per testare le sequenze complete in base alle modifiche apportate dal task

Reply:
Cosa intendi per aggiornare le sequenze in base alle modifiche ? il test menzionato non sarebbe un duplicato di quello committato in quella PR ?